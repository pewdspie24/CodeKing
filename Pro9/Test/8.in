10
4284244452 8393096618
3201470363 6606502723
459690893 1637037427
2075973796 5043157792
2939397287 4964807446
3546569696 6277864456
2157469079 5294748512
4135431913 4803181762
1591597687 4028376886
3476634302 5334077977
