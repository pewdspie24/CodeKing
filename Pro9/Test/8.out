28888172
29759071
12912453
30141665
20148025
22475432
30608425
6438248
25078419
16222708
